Groups/GroupAttendanceMulti.ascx.cs

```diff
@@ -0,0 +1,515 @@
+﻿// <copyright>
+// Copyright 2024 by Kingdom First Solutions
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+//
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data.Entity;
+using System.Linq;
+using System.Web.UI.WebControls;
+
+using Rock;
+using Rock.Attribute;
+using Rock.Data;
+using Rock.Model;
+using Rock.Security;
+using Rock.Web.UI;
+using Rock.Web.UI.Controls;
+
+namespace Plugins.rocks_kfs.Groups
+{
+    [DisplayName( "Multiple Group Attendance" )]
+    [Category( "KFS > Groups" )]
+    [Description( "Allows you to take attendance for multiple groups at once. " )]
+
+    [CustomEnhancedListField( "Groups to Display",
```

Comment:
This setting needs to be higher in the settings block, let's define an order for all block settings.